components/drops/calendar/DropPreviewModal.vue

```diff
@@ -21,11 +21,24 @@
             @click="emit('close')" />
         </header>
 
-        <div v-if="dropCalendar.dropStartTime" class="!mt-6">
-          <NeoButton no-shadow rounded @click="isCreateEventModalActive = true">
+        <div class="!mt-6 flex flex-wrap gap-6">
+          <NeoButton
+            v-if="dropCalendar.dropStartTime"
+            no-shadow
+            rounded
+            @click="isCreateEventModalActive = true">
             {{ $t('scheduled') }}<span class="text-neutral-5 mx-2">•</span
             >{{ formattedDate }}
           </NeoButton>
+
+          <NeoButton
+            v-if="dropCalendar.alias"
+            variant="secondary-rounded"
+            icon-left="sparkles"
+            :tag="NuxtLink"
+            :to="`/ahp/drops/${dropCalendar.alias}`">
```

Comment:
use prefix plz
could also be ahk

Reply:
calendars don't have a `chain` field
 prod /drops only show `ahp` regardless of your chain 

![CleanShot 2024-04-16 at 17 02 26@2x](https://github.com/kodadot/nft-gallery/assets/44554284/4db795ed-b69e-460e-b42e-7c07402f0172)

Reply:
still need to link correctly on canary. don't you agree?

Reply:
Yep

Will add prefix there and chain to calendars

Reply:
fixed https://github.com/kodadot/private-workers/pull/115